homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -26,11 +29,15 @@ def __init__(
         """Initialize AutomowerEntity."""
         super().__init__(coordinator)
         self.mower_id = mower_id
+        entry = coordinator.config_entry
+        structured_token = structure_token(entry.data["token"]["access_token"])
         self._attr_device_info = DeviceInfo(
+            configuration_url=f"{HUSQVARNA_URL}/applications/{structured_token.client_id}",
```

Comment:
I don't think this item is used as intended. This is the url for the application settings, not for the mower device settings. It's not possible to configure the device at this url. Please remove this item.